spec/support/helpers/booking_response_helper.rb

```diff
@@ -1,5 +1,5 @@
 module BookingResponseHelper
-  def reject_visit(visit, reasons = [Rejection::NO_ALLOWANCE])
+  def reject_visit(visit, reasons = [Rejection::SLOT_UNAVAILABLE])
```

Comment:
It's a fait point, although that's rather convenient. The factory itself used to have a default reason
https://github.com/ministryofjustice/prison-visits-2/commit/bc07392b99d1c6088bc3ad73558d9e3a2616828c#diff-23ebfcb2be1d390978dbe12b82767259L4